api-reference/50 Common/utils/ui/notify(message_stack).md

```diff
@@ -0,0 +1,136 @@
+---
+id: ui.notify(message,stack)
+module: ui/notify
+export: default
+---
+---
+##### shortDescription
+Creates a stackable toast message.
+
+##### param(message): String
+The message's text.
+
+##### param(stack): Object
+An object with the following structure: *{position, direction}*.
+
+---
+#include common-demobutton with {
+    url: "https://js.devexpress.com/Demos//WidgetsGallery/Demo/Toast/Stack/"
+}
+
+To stack toast messages, specify the `position` field in the **stack** object.
+
+The `position` field can be one of the following:
+
+- String    
+Available values:
+
+    - *'top left'*
+
+    - *'top center'*
+
+    - *'top right'*
+
+    - *'left center'*
+
+    - *'center'*
+
+    - *'right center'*
+
+    - *'bottom left'*
+
+    - *'bottom center'*
+
+    - *'bottom right'*
+
+- Object    
+Avaliable values: *{left | right, top | bottom}*.
+
+[note]
+
+If you use an object for `position` field, you can only specify one field in that object for vertical coordinate and one field for horizontal coordinate (for example, if you specify *'top'*, you do not need to specify *'bottom'*).
```

Comment:
This explanation sounds a bit strange. I understand that you want to comment on the syntax you used above. But I don't think I would ever try to set a value that reads "right left top". That doesn't make much sense :-) 

On the other hand, I see that "center" values are missing if you specify position via an object. That's maybe something that's worth an emphasis. 

What if you use a single table to list possible values? Then you display all options in one place. You don't need the note. And you also emphasize that String values give you a few extra options. 

| String | Object | 
| ------------- | ------------- |
| `'top left'` | `{left, top}` |
| `'top center'` | N/A |
| `'top right'` | `{right, top}` |
| `'left center'` | N/A |
| `'center'` | N/A |
| `'right center'` | N/A |
| `'bottom left'` | `{left, bottom}` |
| `'bottom center'` | N/A |
| `'bottom right'` | `{right, bottom}` |

This comment applies to the other topic as well.